src/formatDate.js

```diff
@@ -50,7 +50,33 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separator = date.split(fromFormat[fromFormat.length - 1]);
+  const newDate = [];
+
+  const mappedDate = {};
+
+  for (let i = 0; i < fromFormat.length; i++) {
+    if (fromFormat[i] === 'YYYY') {
+      mappedDate['YY'] = separator[i].slice(2);
+    } else if (fromFormat[i] === 'YY') {
+      let year = separator[i];
+
+      year = +year < 30
+        ? 20 + separator[i]
+        : 19 + separator[i];
```

Comment:
Why do you need a `let` variable here ? Let's create cons instead, like I show you in the previous comment